src/linear_algebra/basic.lean

```diff
@@ -1330,11 +1330,14 @@ to_equiv_injective (equiv.eq_of_to_fun_eq h)
 section
 variable (M)
 
+variable (R)
 /-- The identity map is a linear equivalence. -/
 @[refl]
 def refl : M ≃ₗ[R] M := { .. linear_map.id, .. equiv.refl M }
 end
 
+variable {R}
```

Comment:
You can remove this line: the definition of `refl` is made inside a section, so all variables annotations disappear at the end of the section.